content/home/examples/an-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: An Application
+title: Una aplicación
 order: 2
 domid: todos-example
 ---
 
-Using `props` and `state`, we can put together a small Todo application. This example uses `state` to track the current list of items as well as the text that the user has entered. Although event handlers appear to be rendered inline, they will be collected and implemented using event delegation.
+Usando `props` y `state`, podemos integrar todo en una pequeña aplicación de pendientes. Este ejemplo usa `state` para tener un control de la lista actual de objetos así como el texto que el usuario ha entrado. A pesar que los `event handlers` parezcan que fue renderizados, estos seran recolectados e implementados usando una delegación de eventos.
```

Comment:
- I think `ha ingresado` or `introdujo` sounds better than `ha entrado`.
- `fueron` instead of `fue`

Reply:
- I'll go more with `ha ingresado`, but yes you're right `ha entrado` sounds a little odd.